.github/workflows/vercel-beta.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy beta branch to Vercel Preview
+env:
+  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+  VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+on:
+  push:
+    branches:
+      - form-data
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: install Vercel CLI
+        run: npm install --global vercel@latest
+      - name: pull Vercel environment information
+        run: vercel pull --yes --environment=preview --token=${{ secrets.VERCEL_TOKEN }}
+      - name: build project artifacts
+        run: vercel build --token=${{ secrets.VERCEL_TOKEN }}
+      - name: deploy project artifacts to Vercel
+        run: vercel deploy --prebuilt --token=${{ secrets.VERCEL_TOKEN }}
+      - name: assign beta domain
+        run: |
+          vercel deploy --prebuilt --token=${{ secrets.VERCEL_TOKEN }} > domain.txt
+      #    vercel alias --token=${{ secrets.VERCEL_TOKEN }} set `cat domain.txt` beta-linkfree.vercel.app
+      - name: load json files
+        run: curl -f https://linkfree-eddiejaoude-eddiehub.vercel.app/api/system/reload?secret=${{ secrets.LINKFREE_API_SECRET }}
```

Comment:
Is there a way to load `LINKFREE_API_SECRET` using the vercel command?
This would make it easier than keeping the same value in 2 places.

Reply:
Yes true, this could come later as an improvement I think